src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/HKDF.cs

```diff
@@ -108,12 +113,17 @@ public static byte[] Expand(HashAlgorithmName hashAlgorithmName, byte[] prk, int
         /// </summary>
         /// <param name="hashAlgorithmName">The hash algorithm used for HMAC operations.</param>
         /// <param name="prk">The pseudorandom key of at least <see cref="HashLength"/> bytes (usually the output from Expand step).</param>
-        /// <param name="output">The destination buffer to receive the output keying material.</param>
+        /// <param name="output">The destination buffer to receive the output keying material with a size of as least 1.</param>
         /// <param name="info">The context and application specific information (can be an empty span).</param>
+        /// <exception cref="ArgumentException"><paramref name="output"/> length is larger than the allowed length.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="output"/> legnth is less than 1.</exception>
```

Comment:
```suggestion
        /// <exception cref="ArgumentNullException"><paramref name="output"/> is empty.</exception>
```

Reply:
I updated the message (guess I forgot how to spell), but I changed the exception in your suggestion to be `ArgumentOutOfRangeException` since that's what I throw.